packages/bot-web-ui/src/components/flyout.jsx

```diff
@@ -9,6 +9,8 @@ import HelpBase from './help-contents/flyout-help-base.jsx';
 import { connect } from '../stores/connect';
 import '../assets/sass/flyout.scss';
 
+const Hr = () => <div className='terms-of-use__hr terms-of-use__hr__absolute' />;
```

Comment:
Is this component used just for separation?

Reply:
yes, I wanted a line.

Reply:
Can we create a separate css class for it in `bot-web-ui` then, instead of using the one from `core` package?